protocols/identify/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.43.2 - unreleased
+
 ## 0.43.1 - unreleased
```

Comment:
`0.43.1` isn't released yet so we don't need to bump the version again! :)

Reply:
fixed b87b155